### DISCUSSION TITLE
Error using AutoScrollPlugin from @twilio-paste/lexical-library

### DISCUSSION THREAD

**Discussion by noor-ul-amin0** [twilio-labs/paste#3038] [Q&A] [answered]
I'm trying to use the AutoScrollPlugin component from the @twilio-paste/lexical-library package in my React application, but I'm getting the following error: "Minified Lexical error #8; visit https://lexical.dev/docs/error?code=8 for the full message or use the non-minified dev environment for full errors and additional helpful warnings."

I imported the component and added it to my component's render method with a scrollRef prop. I've checked the provided link, but I'm still not sure what's causing the issue. Can someone help me understand what's going wrong and how to fix it? **See line no 74**
![code](https://user-images.githubusercontent.com/56154485/219882065-dbf76898-3372-47d2-b43f-b96eeed8e866.png)

**Comment by nkrantz** [comment, marked as answer]
Hi @noorarman! You're most likely seeing that error because you have the AutoScrollPlugin with the [Chat Log](https://paste.twilio.design/components/chat-log) rather than the [Chat Composer](https://paste.twilio.design/components/chat-composer). It's meant to be used only as a child of the Composer, so that the text the user is typing is always visible. In your case, the plugin is looking for the ChatComposer context, but because it lives with the Log instead of the Composer, it can't find that context. Give that a try and let us know how it goes!
